docs/components/material.md

```diff
@@ -327,42 +327,115 @@ visual defects.
 
 To work around this issue, try changing the order of the entities in the HTML.
 
-## Register a Custom Material
+## Register a Custom Shader Material
 
-We can register custom materials for appearances and effects using
-`AFRAME.registerShader`. However, the `registerShader` API is not yet fully
-featured and fairly limiting (e.g., no `tick` handler, some missing uniform
-types). For most cases for now, we recommend creating a custom material by
-creating three.js materials (e.g., `RawShaderMaterial`, `ShaderMaterial`) in a
-component:
+We can register custom shader materials for appearances and effects using
+`AFRAME.registerShader`.
+
+[example]: https://codepen.io/machenmusik/pen/WZyQNj
+Let's walk through an [example CodePen][example] with step-by-step commentary.
+
+As always, we need to include the A-Frame script.
+Here, we use RawGit to include the latest master version.
 
 ```js
-AFRAME.registerComponent('custom-material', {
+<script src="https://rawgit.com/aframevr/aframe/master/dist/aframe-master.min.js"></script>
```

Comment:
Any reason we need to be suggesting the master version in the docs?

@ngokevin do versions get automatically find/replaced when we do a release?

Reply:
No reason other than not having to change it every release.